docs/release-notes/NuGet-6.10.md

```diff
@@ -191,4 +191,17 @@ Thank you to all the contributors who helped make this NuGet release awesome!
   * [5673](https://github.com/NuGet/NuGet.Client/pull/5673) Enable publishing in VMR
   * [5625](https://github.com/NuGet/NuGet.Client/pull/5625) Eliminate System.CommandLine prebuilt package
 * [brianrob](https://github.com/NuGet/NuGet.Client/pull/5650)
-  * [5650](https://github.com/NuGet/NuGet.Client/pull/5650) Add Restore Instrumentation
\ No newline at end of file
+  * [5650](https://github.com/NuGet/NuGet.Client/pull/5650) Add Restore Instrumentation
+
+## Summary: What's New in 6.10.1
+* [Feature]: add command dotnet config set - [#12476](https://github.com/NuGet/Home/issues/12476)
```

Comment:
Can you please double check this. We normally only fix serious bugs when we service already GA versions. We don't add new features to patch releases.

Reply:
@kartheekp-ms  you mentioned that this was released as part of 6.9 but was missed in the release notes. https://github.com/NuGet/Home/issues/12476#issuecomment-2112985657 

Should I include it in the 6.10.1 release notes?

Reply:
No. If it was released in 6.9, then a PR should update the existing 6.9 release notes.